web/src/features/map/MapWrapper.tsx

```diff
@@ -20,7 +20,13 @@ export default function MapWrapper() {
   return (
     <>
       <MapTooltip />
-      {shouldShowFallback ? <MapFallback /> : <Map />}
+      {shouldShowFallback ? (
+        <MapFallback />
+      ) : (
+        <div className="extend-to-top-edge absolute inset-0">
+          <Map />
+        </div>
```

Comment:
Can this not be handled inside the <Map> component?
We want to avoid creating more elements than is needed.